alica_engine/include/engine/TeamObserver.h

```diff
@@ -27,7 +30,8 @@ class SimplePlanTree;
 class TeamObserver
 {
 public:
-    TeamObserver(AlicaEngine* ae);
+    TeamObserver(Logger& logger, IRoleAssignment& roleAssigment, const bool& maySendMessages, const IAlicaCommunication& communicator, const AlicaClock& clock,
```

Comment:
I think this can be necessary for  `AlicaEngine::reload`
```c++
void AlicaEngine::reload(const YAML::Node& config)
{
    ...
    _maySendMessages = !config["Alica"]["SilentStart"].as<bool>();
   ...
}
```